apps/api/v2/src/config/app.ts

```diff
@@ -1,16 +1,21 @@
+import { getEnv } from "@/env";
+
 import type { AppConfig } from "./type";
 
 const loadConfig = (): AppConfig => {
   return {
     env: {
-      type: (process.env.NODE_ENV as "production" | "development") ?? "development",
+      type: getEnv("NODE_ENV") ?? "development",
```

Comment:
`suggestion-non-blocking`: Fallback could be part of the `getEnv` function. eg: `getEnv("NODE_ENV", "development")`

Reply:
Great idea! Done in [feat: fallback value for getEnv](https://github.com/calcom/cal.com/pull/12677/commits/8a0916668a705df2943231dce39d3713a345a26b)